proxy/api/src/http/error.rs

```diff
@@ -14,6 +14,8 @@ use crate::error;
 pub enum Routing {
     /// The currently active [`coco::User`] is missing.
     MissingOwner,
+    /// The signer is missing.
```

Comment:
The error variant makes sense, but the docs don't reflect it. In general we need to find a representation of three cases: keystore not present, keystore sealed, keystore unsealed.